src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PdfCreator.java

```diff
@@ -37,11 +31,10 @@ public PdfCreator(DuplexPreparator duplexPreparator) {
     private byte[] generatePdf(Document document) {
         Instant start = Instant.now();
         byte[] pdf = generatePdf(document.template.getBytes(), document.values);
-        insights.trackPdfGenerator(Duration.between(start, Instant.now()), true);
         return pdf;
     }
 
-    public PdfDoc create(Letter letter) {
+    public byte[] create(Letter letter) {
```

Comment:
can this be private? if so, maybe just put it in the public method below

Reply:
The reason for the two methods is that the FTP code from the consumer uses the PdfDoc type that saves the files to disk, whereas the Letter entity uses a byte array.